warmup/schemas/postgres/postgres.sql

```diff
@@ -28,6 +28,17 @@ CREATE TABLE IF NOT EXISTS config_search (
   FOREIGN KEY (_id) REFERENCES config (_id) ON DELETE CASCADE ON UPDATE CASCADE
 );
 
+CREATE TABLE IF NOT EXISTS config_metadata (
+  _id varchar(32) NOT NULL,
+  name varchar(64) NOT NULL,
+  value text NOT NULL,
+  FOREIGN KEY (_id) REFERENCES config (_id) ON DELETE CASCADE ON UPDATE CASCADE
```

Comment:
I think both warmup and patch need the unique constraint added. It is quite confusing both exist to be honest

Reply:
Warmup is canonical, patch is to roll up existing users as defined by the schema version number

Reply:
so it does protect against a patch running a second time?

Reply:
In a crude way. Patch files update the schema version as they go.

Something I want to do, but probably will never have time to do, is move this stuff all over to doctrine dbal and have schema definitions stored in json so they can be managed automatically.

Done this for another project and it removes a lot of headaches.